### DISCUSSION TITLE
This is neat !

### DISCUSSION THREAD

**Discussion by shodanx2** [anic17/Batch-Antivirus#13] [General]
For some strange reason, I find myself spending my last few weekends deep in batch file programming with chatgpt as my co-pilot.

I stumbled upon this project from reddit /r/batch 

I am going to look at your code to learn more batch programming strategies for the real world 

thanks !

**Comment by anic17** [comment]
Thanks for your comment! Just to let you know, this isn't by any means a professional antivirus nor it uses modern analysis techniques. Nonetheless, feel free to explore and browse the code, and let me know if you need anything, I'm here to help 😀
